portfolio/src/main/webapp/electronics_info.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The info for each clickable item in the picture
+const info = {
+  tackleBox: 'A cheap fishing tackle-box I bought from a hardware store.\
+      They are the ideal way to organize your small components.',
```

Comment:
Doesn't this end up adding a bunch of spaces in the middle of the text?

I would recommend 'Line 1' + 'Line 2' instead

Reply:
Text inside the <p> tag will have extra spaces removed, which was why I had the JavaScript put this text in a <p> tag. I agree though, your way is less prone to error, and I have fixed it in code.